discovery-infra/test_infra/utils.py

```diff
@@ -58,9 +58,9 @@ def _io_buffer_to_str(buf):
     return out, err, process.returncode
 
 
-def run_command_with_output(command):
+def run_command_with_output(command, env):
```

Comment:
Backwards compatibility not needed?
If so, use empty dict / ``os.environ`` as default value